collection-scripts/gather_network_logs

```diff
@@ -105,6 +105,9 @@ function gather_ovn_kubernetes_nodes_data {
       "ovs-vsctl show" > "${NETWORK_LOG_PATH}"/"${NODE}"_ovs_dump &
       PIDS+=($!)
 
+     oc cp openshift-ovn-kubernetes/"${OVNKUBE_NODE_POD}":/etc/openvswitch/conf.db -c ovnkube-node \
+    "${NETWORK_LOG_PATH}"/"${OVNKUBE_NODE_POD}"_conf.db 2>&1
```

Comment:
@msherif1234 whitespace/indentation looks to be different than existing code?

Reply:
hmm let me check and fix Thanks!!